src/test/java/tr/com/srdc/ontmalizer/test/XSD2OWLTest.java

```diff
@@ -105,6 +114,26 @@ public void writerTest() {
         } catch (Exception e) {
             LOGGER.error("{}", e.getMessage());
         }
-    }
+        exception.expect(NullPointerException.class);   
 
+    }
+    
+    // This part prints the ontology to the specified file.
+      
+    public void xsdTypeCheck(String file) {
```

Comment:
Include this with the assersion to hte tc realted to checking hte extension and remove the helper method